test/extensions/filters/network/http_connection_manager/config_test.cc

```diff
@@ -166,6 +196,9 @@ stat_prefix: router
   request_headers_for_tags:
   - foo
 http_filters:
+- name: envoy.health_check
```

Comment:
Why is this change needed?

Reply:
Not needed - I just wanted the comparable test to have 2 filters in the right order to verify that worked.
Can revert if you'd prefer - obviously we have enough integration tests of multiple filters

Reply:
I'm fine either way. I guess I have a mild preference to revert but I don't feel strongly about it.